pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.luxdb</groupId>
+  <groupId>org.codice</groupId>
   <artifactId>lux</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <version>1.1.1-1-SNAPSHOT</version>
```

Comment:
Change this to 1.1.1-SNAPSHOT. We're in a different group now on our fork, so we don't need to differentiate from the extant 1.1.1-SNAPSHOT version.